src/lib/evil/evil_time.c

```diff
@@ -602,3 +603,28 @@ strptime(const char *buf, const char *fmt, struct tm *tm)
 
    return (char *)bp;
 }
+
+
+#ifdef win32
```

Comment:
Evil is alread used only on windows builds, what you want  here probably is:
```suggestion
#ifdef _MSC_VER
```